### PR TITLE
New package: nullmailer-2.2_1

### PR DESCRIPTION
**Pull request by silum**

### CODE REVIEW

File: srcpkgs/nullmailer/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	# config files
+	rm -f ${wrksrc}/empty
+	touch empty
+	vcopy ${wrksrc}/empty etc/nullmailer/remotes
+
+	# service
+	vsv nullmailer
+}
+
+post_install() {
```
Comment (maxice8) [inline srcpkgs/nullmailer/template]: no need to have a post_install if you have a do_install

File: srcpkgs/nullmailer/INSTALL [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# INSTALL
+case "${ACTION}" in
+post)
+	if [ "${UPDATE}" = "no" ]; then
+		mkfifo -m 600 var/spool/nullmailer/trigger
+		chown _nullmail var/spool/nullmailer/*
+
+		chown _nullmail usr/bin/nullmailer-queue
+		chmod u+s usr/bin/nullmailer-queue
+		chown _nullmail usr/bin/mailq
```
Comment (maxice8) [inline srcpkgs/nullmailer/INSTALL]: /usr/bin/mailq is also claimed by other mail packages which use symlinks.

Create a symlink of /usr/bin/nullmail to /usr/bin/mailq

same apply to sendmail

Comment (maxice8) [reply srcpkgs/nullmailer/INSTALL]: You can look at `dma` `postfix` and `opensmtpd` for examples

Comment (silum) [reply srcpkgs/nullmailer/INSTALL]: `nullmailer` provides a `sendmail` (1) man page.  Should this also be symlinked from, say `nullmailer-sendmail.1`, to `sendmail.1`, or shall I leave it as it is?

File: srcpkgs/nullmailer/template [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	/var/spool/nullmailer/tmp	0700	_nullmail	root"
+makedepends="gnutls-devel"
+short_desc="Simple relay-only mail transport agent"
+maintainer="Deneys S. Maartens <deneys.maartens@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://untroubled.org/nullmailer/"
+distfiles="https://untroubled.org/nullmailer/archive/nullmailer-${version}.tar.gz"
+checksum=9886fa627b7e7e775e2f09fbe789247b1ca304f152da04ff705e50079263a855
+system_accounts="_nullmail"
+conflicts="opensmtpd postfix dma"
```
Comment (maxice8) [inline srcpkgs/nullmailer/template]: Please remove this line.

**Pull request merged**